module/os/linux/zfs/zfs_ioctl_os.c

```diff
@@ -294,9 +294,11 @@ _init(void)
 	zfs_sysfs_init();
 
 	printk(KERN_NOTICE "ZFS: Loaded module v%s-%s%s, "
-	    "ZFS pool version %s, ZFS filesystem version %s\n",
+	    "ZFS pool version %s, ZFS filesystem version %s, "
+	    "ZFS persistent L2ARC version %s\n",
 	    ZFS_META_VERSION, ZFS_META_RELEASE, ZFS_DEBUG_STR,
-	    SPA_VERSION_STRING, ZPL_VERSION_STRING);
+	    SPA_VERSION_STRING, ZPL_VERSION_STRING,
+	    L2ARC_PERSISTENT_VERSION_STRING);
```

Comment:
@behlendorf Agreed. What about reporting specific versions though? In case something is implemented in the future that breaks backwards compatibility of this feature.

Reply:
Thus far, that functionality hasn't been needed but I agree it would be desirable, and possibly necessary, at some point.  It was for this reason that we implemented each feature as its own directory so we could more easily add additional fields, for example a `version` entry.  For the moment, I don't think we need to include the version information but we'll likely want to if there's ever a version 2.